packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/add.controller.js

```diff
@@ -50,7 +52,12 @@ export default class PciServingNamespaceModelsAddController {
 
     this.advancedConfigurationAutoscalerSpec = false;
 
-    [this.model.flavor] = this.flavors;
+    this.frameworks = this.frameworks.filter(
```

Comment:
use lodash filter. It will return an empty array if this.frameworks is null or not defined.